cves/kernel/CVE-2016-6130.yml

```diff
@@ -84,10 +84,6 @@ fixes_instructions: |
 
   Place any notes you would like to make in the notes field.
 fixes:
-- commit:
-  note:
-- commit:
-  note:
 - commit: 532c34b5fbf1687df63b3fcd5b2846312ac943c6
   note: |
```

Comment:
Remove the placeholder note and just have `Manually confirmed` if it is the correct commit.

Reply:
Thank you! I am not sure how I missed that.